packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -897,7 +898,7 @@ class _RenderDecoration extends RenderBox {
       subtextHeight += 8.0;
     return contentPadding.top
       + (label == null ? 0.0 : decoration.floatingLabelHeight)
-      + _lineHeight(width, <RenderBox>[prefix, input, suffix])
+      + _lineHeight(width, <RenderBox>[prefix, prefixIcon, input, suffix])
```

Comment:
This doesn't look right either. We should talk.

Reply:
Fixed